spot_examples/spot_examples/hello_spot.py

```diff
@@ -0,0 +1,224 @@
+import argparse
+import os
+import time
+from typing import Optional
+
+import bosdyn.geometry
+import cv2
+import synchros2.process as ros_process
+import synchros2.scope as ros_scope
+from bosdyn.api import trajectory_pb2
+from bosdyn.api.spot import robot_command_pb2 as spot_command_pb2
+from bosdyn.client import math_helpers
+from bosdyn.client.frame_helpers import GRAV_ALIGNED_BODY_FRAME_NAME, ODOM_FRAME_NAME
+from bosdyn.client.robot_command import RobotCommandBuilder
+from bosdyn.util import seconds_to_duration
+from bosdyn_msgs.conversions import convert
+from cv_bridge import CvBridge
+from rclpy.node import Node
+from sensor_msgs.msg import Image
+from synchros2.action_client import ActionClientWrapper
+from synchros2.tf_listener_wrapper import TFListenerWrapper
+from synchros2.utilities import namespace_with
+
+from spot_msgs.action import RobotCommand  # type: ignore
+
+from .simple_spot_commander import SimpleSpotCommander
+
+
+class HelloSpot:
+    def __init__(self, robot_name: Optional[str] = None, node: Optional[Node] = None) -> None:
+        self.node = ros_scope.node()
+        if self.node is None:
+            raise ValueError("no ROS 2 node available (did you use bdai_ros2_wrapper.process.main?)")
+        self.logger = self.node.get_logger()
+
+        self.image_sub = self.node.create_subscription(
+            Image, f"/{robot_name}/camera/frontleft/image", self.image_callback, 10
```

Comment:
nit:
```suggestion
            Image, namespace_with(robot_name, "camera/frontleft/image"), self.image_callback, 10
```
I think this will make it work if robot_name is not specified (i.e. if the driver is launched without a namespace)

Reply:
Ah ok, great idea. Thanks!